### PR TITLE
Entity Pages

### PR DESCRIPTION
**Pull request by cesarvarela**
@smcgregor a preliminary version: 

https://deploy-preview-985--staging-aiid.netlify.app/entities

### CODE REVIEW

**Comment by cesarvarela** [comment]
@smcgregor this is ready for another review

- added Common Entities card to landing page
- clickable harmed parties on citation pages
- small fixes

**Comment by pdcp1** [comment]
@cesarvarela This is more of a design comment, but maybe it's better to have the `Learn More` buttons aligned at the bottom of the card. A possible solution could be to have a fixed Card height.

image

File: site/gatsby-site/src/templates/entity.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+                      {entity[section.key].length > 3 && (
+                        <button
+                          onClick={() => setOpen((open) => !open)}
+                          className="text-blue-700 border hover:bg-blue-700 hover:text-white focus:ring-4 focus:outline-none focus:ring-blue-300 rounded-lg text-xs p-1.5 text-center inline-flex items-center mr-2  dark:text-blue-500 dark:hover:text-white dark:focus:ring-blue-800"
+                        >
+                          {open ? (
+                            <Trans>View Less</Trans>
+                          ) : (
+                            <Trans>View ({{ hidden }}) more</Trans>
```
Comment (pdcp1) [inline site/gatsby-site/src/templates/entity.js:172]: Add missing translation, these two are not working in Spanish

File: site/gatsby-site/src/templates/entity.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+
+                          return (
+                            <li key={incident.incident_id} className="py-3 sm:py-4">
+                              <Link
+                                to={`/cite/${incident.incident_id}`}
+                                className="tracking-tight text-gray-900 dark:text-white"
+                              >
+                                <div className="m-0">
+                                  <div className="inline-block text-muted-gray">
+                                    Incident {incident.incident_id}
```
Comment (pdcp1) [inline site/gatsby-site/src/templates/entity.js:151]: Add missing translation

File: site/gatsby-site/src/templates/entity.js [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+                                  </div>
+                                  <div className="ml-2 inline-block bg-red-100 text-red-800 text-xs font-semibold mr-2 px-2.5 py-0.5 rounded dark:bg-green-200 dark:text-green-900">
+                                    {incident.reports.length} Reports(s)
```
Comment (pdcp1) [inline site/gatsby-site/src/templates/entity.js:154]: Add missing translation

File: site/gatsby-site/src/templates/entity.js [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+                .map((section) => {
+                  const [open, setOpen] = useState(false);
+
+                  const visible = 3;
+
+                  const hidden = entity[section.key].length - visible;
+
+                  return (
+                    <Fragment key={section.key}>
+                      <h5 className="mt-6">{section.header}</h5>
```
Comment (pdcp1) [inline site/gatsby-site/src/templates/entity.js:136]: Add missing translation

File: site/gatsby-site/src/components/entities/AllegedEntities.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    entitiesHarming.every(
+      (e) => e.incidentsAsDeveloper.length == 0 && e.incidentsAsDeployer.length === 0
+    )
+  ) {
+    return (
+      <Trans>
+        Alleged:{' '}
+        <EntitiesList entities={entitiesHarming.length ? entitiesHarming : entitiesHarmed} />{' '}
+        developed and deployed an AI system, which harmed <PartiesList entities={entitiesHarmed} />.
+      </Trans>
```
Comment (pdcp1) [inline site/gatsby-site/src/components/entities/AllegedEntities.js:64]: This translation is not working. Is it missing on the translation files?

File: site/gatsby-site/src/components/entities/AllegedEntities.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+  const developers = entitiesHarming.filter(
+    (e) => e.incidentsAsDeveloper.length > 0 || e.incidentsAsBoth.length > 0
+  );
+
+  return (
+    <Trans>
+      Alleged: <EntitiesList entities={developers} /> developed an AI system deployed by{' '}
+      <EntitiesList entities={deployers} />, which harmed <PartiesList entities={entitiesHarmed} />.
+    </Trans>
```
Comment (pdcp1) [inline site/gatsby-site/src/components/entities/AllegedEntities.js:80]: This translation is not working. Is it missing from the translation files?

**Review by pdcp1 — CHANGES_REQUESTED** [review]

**Comment by cesarvarela** [comment]
Lol looks like halfway through, I decided translations were not worth any more xd, fixed.

I'm unsure what to do about the cards; both options will look weird; I could add a "read more" button to make the descriptions have the same length, but I wouldn't say I like it much.

**Comment by cesarvarela** [comment]
@smcgregor please give it another look, it is almost ready to merge to staging

**Review by pdcp1 — APPROVED** [review]

**Pull request merged**